src/pybindings/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.1)
+
+if(NOT FAST_DOWNWARD_MAIN_CMAKELISTS_READ)
+    message(
+        FATAL_ERROR
+        "Run cmake on the CMakeLists.txt in the 'src' directory, "
+        "not the one in 'src/search'. Please delete CMakeCache.txt "
+        "from the current directory and restart cmake.")
+endif()
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+## == Project ==
+
+# TODO: Does that need to be project(pydownward) or project(downward)?
+project(pydownward)
+
+# TODO: use multi-line strings to break up the long string when we switch to CMake 3.
+# https://stackoverflow.com/questions/7637539/how-to-split-strings-across-multiple-lines-in-cmake
+option(
+  USE_GLIBCXX_DEBUG
+  "Enable the libstdc++ debug mode that does additional safety checks. (On Linux systems, g++ and clang++ usually use libstdc++ for the C++ library.) The checks come at a significant performance cost and should only be enabled in debug mode. Enabling them makes the binary incompatible with libraries that are not compiled with this flag, which can lead to hard-to-debug errors."
+  FALSE)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+ 
+# TODO: Can you use the same flags here as in search/CMakeLists.txt?
+# If yes, uncomment following two and delete the subsequent two.
+# fast_downward_set_compiler_flags()
+# fast_downward_set_linker_flags()
+set(CMAKE_CXX_FLAGS "-O3")
+set(CMAKE_CXX_FLAGS_RELEASE "-O3")
+
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../search/ext/")
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../search/")
+
+# TODO: Need to integrate DownwardFiles.cmake
+file (GLOB SOURCE_FILES "../search/*.cc" "../search/*/*.cc")
```

Comment:
`GLOB` is a hack that we should avoid. The file `src/search/DownwardFiles.cmake` defines the list of sources and `search/CMakeLists.txt` already collects them and handles the dependencies.

Reply:
We could maybe treat the file that defines the interface (currently called `test.cc` but we have to find a better name) as a CMake plugin. Then the build configuration for the library would enable this plugin and all other configs would disable it.